bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -224,7 +225,7 @@ def rotate_gripper_with_delta(self, wrist_yaw=0.0, wrist_roll=0.0):
         self.set_arm_joint_positions(new_arm_joint_states)
         time.sleep(0.5)
 
-    def move_gripper_to_point(self, point, rotation):
+    def move_gripper_to_point(self, point, rotation, seconds_to_goal=3.0):
```

Comment:
Does the 3 second constraint on time for trajectory look good? @jimmytyyang  @akshararai

Reply:
This could move to the yaml file! But I think 3 seconds looks good!

Reply:
Good idea.
Another clarification, the default value is 3sec. But for our case in place env, we were using 1 second as the time.


In the config, I will make the time as 2secs so as to have smoother motion.